programs/magicblock/src/magic_scheduled_base_intent.rs

```diff
@@ -316,7 +340,17 @@ impl CommitType {
         context: &ConstructionContext<'_, '_>,
     ) -> Result<(), InstructionError> {
         accounts.iter().try_for_each(|(pubkey, account)| {
-            let owner = *account.borrow().owner();
+            let account_shared = account.borrow();
+            if !account_shared.delegated() {
+                ic_msg!(
+                    context.invoke_context,
+                    "ScheduleCommit ERR: account {} has needs to be delegated to current validator to be committed",
```

Comment:
typo: better wording would be `is required to be delegated to the current validator, in order to be committed`

Reply:
Done a9eb14655364cadd4c46301657a737802fa490e2